examples/multi_window_ref_app/lib/app/main_window.dart

```diff
@@ -38,162 +33,166 @@ class _MainWindowState extends State<MainWindow> {
+              DataColumn(
+                label: SizedBox(
+                  width: 20,
+                  child: Text(
+                    'ID',
+                    style: TextStyle(
+                      fontSize: 16,
+                    ),
+                  ),
+                ),
               ),
-            ),
-          ),
-        ),
-        DataColumn(
-          label: SizedBox(
-            width: 120,
-            child: Text(
-              'Type',
-              style: TextStyle(
-                fontSize: 16,
+              DataColumn(
+                label: SizedBox(
+                  width: 120,
+                  child: Text(
+                    'Type',
+                    style: TextStyle(
+                      fontSize: 16,
+                    ),
+                  ),
+                ),
               ),
-            ),
-          ),
-        ),
-        DataColumn(
-            label: SizedBox(
-              width: 20,
-              child: Text(''),
-            ),
-            numeric: true),
-      ],
-      rows: windows.asMap().entries.map<DataRow>((indexedEntry) {
-        final index = indexedEntry.key;
-        final Window entry = indexedEntry.value;
-        final window = entry;
-        final viewId = window.view.viewId;
-        final archetype = window.archetype;
-        final isSelected = selectedRowIndex == index;
-
-        return DataRow(
-          color: WidgetStateColor.resolveWith((states) {
-            if (states.contains(WidgetState.selected)) {
-              return Theme.of(context).colorScheme.primary.withOpacity(0.08);
-            }
-            return Colors.transparent;
-          }),
-          selected: isSelected,
-          onSelectChanged: (selected) {
-            if (selected != null) {
-              onSelectedRowIndexChanged(selected ? index : -1);
-            }
-          },
-          cells: [
-            DataCell(
-              Text('$viewId'),
-            ),
-            DataCell(
-              Text(archetype.toString().replaceFirst('WindowArchetype.', '')),
-            ),
-            DataCell(
-              IconButton(
-                icon: const Icon(Icons.delete_outlined),
-                onPressed: () {
-                  destroyWindow(context, window);
+              DataColumn(
+                  label: SizedBox(
+                    width: 20,
+                    child: Text(''),
+                  ),
+                  numeric: true),
+            ],
+            rows: (windowManagerModel.windows)
+                .map<DataRow>((KeyedWindowController controller) {
+              return DataRow(
+                key: controller.key,
+                color: WidgetStateColor.resolveWith((states) {
+                  if (states.contains(WidgetState.selected)) {
+                    return Theme.of(context)
+                        .colorScheme
+                        .primary
+                        .withOpacity(0.08);
```

Comment:
```suggestion
                        .withAlpha(20);
```
`withOpacity` has been deprecated.